scantool/scantool_cli.c

```diff
@@ -434,6 +436,7 @@ cmd_log(int argc, char **argv) {
 	char autofilename[20]="";
 	char *file;
 	time_t now;
+	char timestr[26];
```

Comment:
is it [26] here on purpose ? (the previous strftime had [256] ) ? I think 256 is more than enough, but 26 may be a bit short. Most languages/locales can probably fit in 80 I guess ?

Reply:
No this is rubbish - corrected.